integration/jnifuse/native/src/main/native/libjnifuse/jnifuse_impls.cc

```diff
@@ -14,6 +14,19 @@
 #include "debug.h"
 #include "jnifuse_fs.h"
 
+void* init_wrapper(struct fuse_conn_info* conn) {
+  #ifndef __APPLE__
+  if((unsigned int)conn->capable & FUSE_CAP_ATOMIC_O_TRUNC){
+      conn->want |= FUSE_CAP_ATOMIC_O_TRUNC;
+  }
+  #endif
+
+  if((unsigned int)conn->capable & FUSE_CAP_BIG_WRITES){
+    conn->want |= FUSE_CAP_BIG_WRITES;
+  }
+  return NULL;
```

Comment:
always return a NULL?

Reply:
https://libfuse.github.io/doxygen/structfuse__operations.html#a3a1f9562a38871caaf4fc2de7ea5b470

It should be fine as long as it is only called once.